yt_dlp/extractor/common.py

```diff
@@ -314,6 +314,11 @@ class InfoExtractor:
                         * "author" - human-readable name of the comment author
                         * "author_id" - user ID of the comment author
                         * "author_thumbnail" - The thumbnail of the comment author
+                        * "author_url" - The url to the comment author's page
+                        * "author_is_verified" - Whether the author is verified
```

Comment:
Not sure if author_is_verified is worth a field.

Reply:
I think its nice to have and is general across other sites too.

Plus I'm adding `channel_is_verified` in a following PR so might as well have it here as well.